### PR TITLE
Create landing page template

### PR DESCRIPTION
**Pull request by brikis98**
This is a dirt simple template we can copy/paste to create quick landing pages for the [market test described here](https://www.notion.so/gruntwork/Market-testing-417edc6f39fa422ca22d0cd7357257bf). 

You can experiment with the template here: https://deploy-preview-452--keen-clarke-470db9.netlify.app//fill-in-a-link-here/

### CODE REVIEW

File: pages/contact/_form.html [hunk 1]
```diff
@@ -12,7 +12,8 @@
     value="New inquiry from {{ site.url }}"
   />
   <input type="hidden" name="_viewing" id="_viewing" value="v1-static" />
-  <input type="hidden" name="_next" value="{{ site.url }}/thanks/" />
+  <input type="hidden" name="_next" id="_next" value="{{ site.url }}/thanks/" />
+  <input type="hidden" name="hello_gruntwork" value=""/>
```
Comment (brikis98) [inline pages/contact/_form.html:16]: We have a "honey pot" configured in FormBucket, but we didn't actually put the form element in our forms, so no bot would ever be tripped up by it...

File: assets/js/main.js [hunk 2]
```diff
@@ -8553,33 +8554,36 @@ $(function() {
...
   };
 
-  var submitToFormSpree = function(data) {
+  var submitToFormSpree = function(url, data, redirect) {
     submitButton.html("Sending...");
     submitButton.prop("disabled", true);
 
     var postParams = {
-      url: "https://api.formbucket.com/f/buk_7iB8j7vEJPW9ad2ClJwFfm5M",
+      url: url,
```
Comment (brikis98) [inline assets/js/main.js:8586]: I've updated our form submission code to read the URL from the `action` of the form, so we can use the same code with multiple forms. The redirect URL is configurable too, via the `_next` param.

Comment (robmorgan) [reply assets/js/main.js:8586]: Cool!

File: pages/contact/_form.html (hunk 1 above)
Comment (robmorgan) [reply pages/contact/_form.html:16]: Nice find!

**Comment by robmorgan** [comment]
@brikis98 is there any point merging this PR to `master` or does it just serve as an example? I'm going to checkout a branch off this now.

**Comment by brikis98** [comment]
> @brikis98 is there any point merging this PR to `master` or does it just serve as an example? I'm going to checkout a branch off this now.

Perhaps for future experiments? Not sure. Creating a branch off this sounds good for now.

**Comment by brikis98** [comment]
Closing, as this content was merged to master in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/453.

**Pull request closed without merging**